configs/textrecog/sar/sar_r31_parallel_decoder_chineseocr.py

```diff
@@ -0,0 +1,128 @@
+_base_ = ['../../_base_/default_runtime.py']
+
+dict_file = 'data/chineseocr/labels/dict_printed_chinese_english_digits.txt'
```

Comment:
This dict labels/dict_printed_chinese_english_digits.txt is not released. How to decode correctly via others?